translations/Hindi/special-terms.md

```diff
@@ -0,0 +1,179 @@
+
+# Terms
+
+* Message: संदेश
+
+  Message has a direct translation in hindi
+
+  टिम(Tim) ने आप एक प्राइवेट संदेश भेजा है।
+* Private Message (PM): निजी संदेश
+
+  निजी is the direct translation for Private in Hindi.
+
+  टिम(Tim) ने आप एक निजी संदेश भेजा है।
+* Group PM: समूह निजी संदेश
+
+  Group and message have direct translations in Hindi.
+
+  टिम(Tim) ने आप और राम के लिए एक समूह निजी संदेश भेजा है।
+
+* Realm: संगठन / टीम
+
+  Realm has a word-to-word translation of क्षेत्र in Hindi, but क्षेत्र
+  directly translates to Province in English which is clearly inappropriate for
+  Zulip as Realm in Zulip documents is interpreted as “Organization”.
+  Therefore, I propose the use of Organization or Team(direct word translation
+  in Hindi) instead. IMO using Team will be more appropriate.
+
+  आप ने एक संगठन/टीम बनाया है।
+
+* Stream: धारा
+
+  Direct translation of the word "Stream". I thought of different translations
+  like चैनल but that doesn't fit will with the Zulip context.
+
+  आप को बैकएंड धारा में सूचनाओं की है।
+
+* Topic: विषय
+
+  This is a direct translation and fits perfectly.
+
+  आप एक विषय के बिना कोई धारा पे संदेश नहीं भेज सकते हैं।
+
+* Private/Invite-Only Stream: प्राइवेट धारा
+
+  निजी is a direct translation for Private and fits well in Zulip's context,
+  therefore प्राइवेट धारा seems apt for this.
+
+  आप को एक निजी धारा मे आमंत्रित किया गया है ।
+
+* Public Stream: सार्वजनिक धारा
+
+  सार्वजनिक isn't the direct translation for Public, जनता is, but this fits
+  well in Zulip's context.
+
+  आप backend सार्वजनिक धारा के सदस्य हैं ।
+* Bot: रोबोट
+
+  रोबोट means Robot in Hindi but it should fit fine in Zulip's context.
+
+  आप को Github रोबोट से एक संदेश मिल​ है ।
+
+* Integration: समाकलन
+
+  The direct translation of Integration is एकीकरण which translates to "joining"
+  which isn't idea for Zulip. I feel समाकलन will be a better fit.
+
+  GitHub के लिए एक समाकलन बनाने ।
+* Notification: अधिसूचना
+
+  Direct translation.
+
+  सूचनाएं दिखाने।
+* Alert Word: चेतावनी
+
+  Direct translation.
+
+  चेतावनी सेटिंग्स संपादित करे
+
+* View: HELP NEEDED
+
+* Home: घर
+  Direct translation.
+  घर ले चलो
+
+* Emoji: इमोजी
+  HELP NEEDED
+
+# Phrases
+
+* Subscribe to a stream/Unsubscribe from a stream: इस धारा की सदस्यता करे/ इस
```

Comment:
इस धारा की सदस्यता प्राप्त करे
I am unable to write prapt as we do in Hindi, it should not be प्रा but that other alphabet.
I hope you can understand.
Try if you find it else, everywhere प्रा is used. I think it will go.